samples/snippets/src/main/java/com/example/spanner/HubbleTransactionsCodeLab.java

```diff
@@ -34,31 +37,40 @@ public void createInterleaved(DatabaseAdminClient dbAdminClient, DatabaseId id)
         id,
         Arrays.asList(
             "CREATE TABLE Mailbox("
-                + " sid INT64 NOT NULL,"
-                + " state STRING(MAX),"
+                + "sid INT64 NOT NULL,"
+                + "guid STRING(MAX),"
+                + "state STRING(MAX),"
                 + ") PRIMARY KEY (sid)",
             "CREATE TABLE Message("
                 + " sid INT64 NOT NULL,"
                 + " msg_id STRING(MAX) NOT NULL,"
+                + " subject STRING(MAX),"
                 + " body STRING(MAX),"
+                + " send_timestamp TIMESTAMP  OPTIONS (allow_commit_timestamp=true),"
                 + ") PRIMARY KEY (sid, msg_id),"
-                + " INTERLEAVE IN PARENT Mailbox ON DELETE CASCADE"));
+                + "INTERLEAVE IN PARENT Mailbox ON DELETE CASCADE"));
+
+    System.out.println("Finished execution of createInterleaved()");
   }
 
   public void createWorkItems(DatabaseAdminClient dbAdminClient, DatabaseId id) {
     createDatabase(
         dbAdminClient,
         id,
-        Arrays.asList(
+        Collections.singletonList(
             "CREATE TABLE WorkList("
                 + " id STRING(MAX) NOT NULL,"
                 + " is_done BOOL,"
+                + " generated_value Int64,"
```

Comment:
Why do we need this?

Reply:
generated_value is incremental, this is added to get a sequence  number for rows. Later on for Locking use case I am using query to filter based on generated value. Its easy to distribute the row between threads using it, so instead of limit clause I am using this value in the query in where clause for different threads.

Reply:
This doesn't seem like what we should be doing. This is basically a monotonically increasing index that hasn't been added as a key. Adding it as a key makes a monotonically increasing index on the table, which is bad. Not adding it as a key lets there be duplicate values in the field, which will make it not work for what you are using it for, I believe. We should remove this.